RefactoringsExperiments/src/main/java/ru/hse/kirilenko/refactorings/utils/calcers/KeywordsCalculator.java

```diff
@@ -37,4 +39,19 @@ public static void calculateCSV(String codeFragmentString, int fragLinesCount) {
             id += 2;
         }
     }
+
+    public static void extractToList(String codeFragmentString, List<ICSVItem> features, int fragLinesCount) {
+        HashMap<String, Integer> counts = KeywordsCalculator.keywordsTrie.calculate(codeFragmentString);
+        int id = 6;
```

Comment:
why 6?

Reply:
because first keyword feature has `id = 6`

Reply:
(that is how it was prior to my changes as well)

Reply:
ok, add a comment about it